.github/workflows/multi-node-test-workflow.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           echo "Running backwards compatibility tests ..."
           chown -R 1000:1000 `pwd`
-          su `id -un 1000` -c "./gradlew bwcTestSuite -Dtests.security.manager=false"
+          su `id -un 1000` -c "./gradlew bwcTestSuite -Dtests.security.manager=false -PcustomDistributionDownloadType=bundle"
```

Comment:
Does this increase run time at all?

Reply:
Bundle level tests removed as not appropriate for repo workflows.